cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -108,6 +109,19 @@ message GetAvailablePackageVersionsRequest {
   string pkg_version = 2;
 }
 
+// GetInstalledPackageSummariesRequest
+//
+// Request for GetInstalledPackageSummeraies
+message GetInstalledPackageSummariesRequest {
+  // The context (cluster/namespace) for the request.
+  Context context = 1;
+
```

Comment:
We don't currently support filtering of installed packages at all, so I've removed the TODO from the design doc for here in the request (and in the doc).